src/acceptance-test/java/de/zalando/aruha/nakadi/webservice/UserJourneyAT.java

```diff
@@ -19,7 +19,7 @@
 
 public class UserJourneyAT extends RealEnvironmentAT {
 
-    private static final String TEST_EVENT_TYPE = randomString();
+    private static final String TEST_EVENT_TYPE = "order.ORDER_CANCELLED";
```

Comment:
such a name will limit the possibility to run this UserJourney test on a real system (because it can conflict with real name)
Maybe `randomString() + "." + randomString()` ?